Problem: build ra-sp-server failed

```
$ cargo build -p ra-sp-server
   Compiling ra-common v0.5.0 (/Users/yihuang/src/chain/chain-tx-enclave/enclave-ra/ra-common)
   Compiling ra-sp-server v0.5.0 (/Users/yihuang/src/chain/chain-tx-enclave/enclave-ra/ra-sp-server)
error[E0271]: type mismatch resolving `<[u8; 4] as std::convert::TryFrom<std::vec::Vec<u8>>>::Error == std::array::TryFromSliceError`
  --> chain-tx-enclave/enclave-ra/ra-sp-server/src/context.rs:59:14
   |
59 |             .try_into()
   |              ^^^^^^^^ expected enum `std::convert::Infallible`, found struct `std::array::TryFromSliceError`

error[E0277]: the trait bound `[u8; 4]: std::convert::From<std::vec::Vec<u8>>` is not satisfied
  --> chain-tx-enclave/enclave-ra/ra-sp-server/src/context.rs:59:14
   |
59 |             .try_into()
   |              ^^^^^^^^ the trait `std::convert::From<std::vec::Vec<u8>>` is not implemented for `[u8; 4]`
   |
   = note: required because of the requirements on the impl of `std::convert::Into<[u8; 4]>` for `std::vec::Vec<u8>`
   = note: required because of the requirements on the impl of `std::convert::TryFrom<std::vec::Vec<u8>>` for `[u8; 4]`
   = note: required because of the requirements on the impl of `std::convert::TryInto<[u8; 4]>` for `std::vec::Vec<u8>`

error[E0061]: this function takes 5 arguments but 6 arguments were supplied
  --> chain-tx-enclave/enclave-ra/ra-sp-server/src/context.rs:74:14
   |
74 |             .get_quote(
   |              ^^^^^^^^^ expected 5 arguments
75 |                 &self.quote_info,
   |                 ----------------
76 |                 report,
   |                 ------
77 |                 self.spid.to_vec(),
   |                 ------------------
78 |                 sig_rl,
   |                 ------
79 |                 parse_quote_type(&self.quote_type)?,
   |                 -----------------------------------
80 |                 nonce.to_vec(),
   |                 -------------- supplied 6 arguments

error: aborting due to 3 previous errors
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -35,6 +35,7 @@ rust: &rust
         - binutils-dev
         - libc6-dev
         - libzmq3-dev
+        - protobuf-compiler
   cache:
     directories:
       - $HOME/.cargo

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -46,6 +46,7 @@ default-members = [
     "test-common",
     "dev-utils",
     "enclave-protocol",
+    "chain-tx-enclave/enclave-ra/ra-sp-server",
     "chain-tx-enclave/enclave-utils",
     "chain-tx-enclave/mls",
     "cro-clib",

--- a/chain-tx-enclave/enclave-ra/ra-sp-server/src/context.rs
+++ b/chain-tx-enclave/enclave-ra/ra-sp-server/src/context.rs
@@ -56,6 +56,7 @@ impl SpRaContext {
         let gid = self
             .quote_info
             .gid()
+            .as_slice()
             .try_into()
             .map_err(SpRaContextError::InvalidGid)?;
 
@@ -72,7 +73,6 @@ impl SpRaContext {
         let quote_result = self
             .aesm_client
             .get_quote(
-                &self.quote_info,
                 report,
                 self.spid.to_vec(),
                 sig_rl,

--- a/docker/build.sh
+++ b/docker/build.sh
@@ -29,6 +29,7 @@ else
     cargo build $CARGO_ARGS --features mock-enclave --manifest-path client-cli/Cargo.toml
     cargo build $CARGO_ARGS --features mock-enclave --manifest-path dev-utils/Cargo.toml
     cargo build $CARGO_ARGS --features mock-enclave --manifest-path chain-abci/Cargo.toml
+    cargo build $CARGO_ARGS -p ra-sp-server
 fi
 
 echo "Build dynamic cro-clib"

